unit/java_bytecode/java_bytecode_parse_generics/GenericClassWithNestedGenericInnerClasses.java

```diff
@@ -0,0 +1,7 @@
+public class GenericClassWithNestedGenericInnerClasses<T> {
+  class Generic2<U> {
+    class Generic3<V> {
+    }
+  }
+  Generic2<Integer>.Generic3<String> f;
+}
```

Comment:
Nit: files should end with a new line ([sublime can be configured](https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842) to auto add these on save, as can other editors I assume). GH shows this as a little red icon at the end of the file: :no_entry_sign: